http/src/main/resources/reference.conf

```diff
@@ -194,7 +194,7 @@ azure {
         type = "CustomScript",
         version = "2.1",
         minor-version-auto-upgrade = true,
-        file-uris = ["https://raw.githubusercontent.com/DataBiosphere/leonardo/d6f5eea1a9299f8ef95cf1dc3eaf40594af26782/http/src/main/resources/init-resources/azure_vm_init_script.sh"]
+        file-uris = ["https://raw.githubusercontent.com/DataBiosphere/leonardo/24d19970c5785b5e8f94d7660dcc8a0b4c497ff9/http/src/main/resources/init-resources/azure_vm_init_script.sh"]
```

Comment:
So Leo will always use the `azure_vm_init_script.sh` from this git hash. It looks like https://raw.githubusercontent.com/DataBiosphere/leonardo/24d19970c5785b5e8f94d7660dcc8a0b4c497ff9/http/src/main/resources/init-resources/azure_vm_init_script.sh is an older version which doesn't have the changes in this branch. 

I think you want https://raw.githubusercontent.com/DataBiosphere/leonardo/ad6056757f684e2bd82fc1d90240dab7db333862/http/src/main/resources/init-resources/azure_vm_init_script.sh to point at the current version in this branch. Or even better just use https://github.com/DataBiosphere/leonardo/blob/IA-3957-include-additional-create-parameters/http/src/main/resources/init-resources/azure_vm_init_script.sh to have it track the branch (just change back to hash before merge).

Reply:
Also this setup is a little odd, feels a bit error-prone and brittle to use direct references to git from Leo. There may be another way to do this (but separate from this PR).

Reply:
Huh. When I tried to do this before, I couldn't see my changes on the browser. It seemed like it was taking it's time to update the 'raw' file. 🤷 Seems fine now.